catalogue/webapp/pages/search/images.tsx

```diff
@@ -57,6 +58,8 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
     setLink(link);
   }, [imagesRouteProps]);
 
+  const showSort = true;
```

Comment:
is this going to be set some other way in the future?

Reply:
That is removed in one of my following PRs

Reply:
@AMBacelar am I OK to merge it?

Reply:
Indeed, I'll have that resolved in the sort PR